editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Question.tsx

```diff
@@ -80,9 +82,18 @@ const Question: React.FC<Props> = React.memo((props) => {
           onContextMenu={handleContext}
           ref={drag}
         >
+          {props.data?.img && (
+            <Thumbnail
+              imageSource={props.data?.img}
+              imageAltText={props.data?.text}
+            />
+          )}
           {Icon && <Icon titleAccess={iconTitleAccess} />}
           <span>{props.text}</span>
         </Link>
+        {props.type !== TYPES.SetValue && props.data?.fn && (
+          <DataField value={props.data.fn} variant="parent" />
+        )}
```

Comment:
`props.data.fn` will _not_ catch all data fields on nodes (eg especially individual file types in FileUpload&Label, List component, etc) but it should catch all the major "top-level" data fields - which feels "good enough" for this experiment! 

Would be a nice exercise to apply some of Daf's thorough search mappings here in the future for more accurate coverage.